modules/test/conn/python/src/connection_module.py

```diff
@@ -160,6 +214,12 @@ def _connection_single_ip(self):
     else:
       return result, 'Device is using multiple IP addresses'
 
+  def _get_dhcp_type(self, packet):
+    for option in packet[DHCP].options:
+      # message-type, option 3 = DHCPREQUEST

```

Comment:
Comment appears to be more aligned with the change on line 198 and doesn't fit here.